test/global_state_test.ts

```diff
@@ -261,18 +397,19 @@ function replayMsgs() {
   //assert(maxUserID < maxAccountNum);
   let accounts: Array<Account> = [];
   for (let i = 0; i < maxAccountNum; i++) {
-    // currently the matchengine generates order id from 1
-    const accountID = state.createNewAccount({ next_order_id: 1n });
+    //next order pos has no relation with order id
+    const accountID = state.createNewAccount({ next_order_id: 0n });
     assert(accountID == BigInt(i), 'invalid account id');
-    accounts[i] = Account.random();
+    // accounts[i] = Account.random();
     //  for (let j = 0; j < maxTokenNum; j++) {
     //    state.setTokenBalance(accountID, BigInt(j), 1_000_000n); // default balance
     //  }
   }
   for (const msg of msgs) {
-    if (msg.type === 'BalanceMessage') {
+    // DepositMessage is new version for BalanceMessage
+    if (msg.type === 'BalanceMessage' || msg.type == 'DepositMessage') {
```

Comment:
We should use `if (msg.type === 'BalanceMessage' || msg.type == 'DepositMessage')`

Because for now "deposit" "withdraw" and "trade" will all generate "balance" message.

Reply:
Old version:
only "BalanceMessage". change > 0 is deposit. change < 0 is withdraw. 

New version:
"DepositMessage" "WithdrawMessage".